source/tools/hubspot.rst

```diff
@@ -0,0 +1,111 @@
+Hubspot
+============
+
+What is Hubspot?
+----------------
+HubSpot is an "inbound marketing software platform" which offers features for internet marketing, such as emails, blogging and social media. This piece of software is mostly used by the marketing team.
+
+This document will mostly cover tips on how to use the emailer tool/rehabstudio email template.
+
+The rehabstudio email template
+------------------------------
+
+The 2 in 1 template allows you to switch between the event or newsletter email layouts. Each layout includes a spotlight, monthly links and footer section. The spotlight and monthly links can be hidden if not needed.
+
+Name of template:  rehabstudio_new_mailer.html
+
+Getting started on hubspot
+--------------------------
+
+Create a new email using the hubspot template
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To create a new email with Hubspot.
+
+- Click create new email.
+- Select the new template titled rehabstudio_new_mailer.html.
+- Give the email a title, then click create.
+
+The Editor
+------------------
+
+This section allows you to edit the contents of the email template. Each of the contents are stored in modules. You can manipulate the modules by showing or hiding them.
+
+The rehabstudio_new_mailer template has a number of custom modules, with options to either show or hide sections. This will depend on the type of email your building. The options can be found in the ‘Edit Modules’ panel and are highlighted in orange.
+
+Important! - you may have to refresh the page to see changes within the edit module panel and template.
+
+Choose type of email
+~~~~~~~~~~~~~~~~~~~~
+This option will determine if your making either an Event or an Newsletter. Selecting one will change the layout of the email, opening more options for further editing.
+
+- Event
+    Display event layout.
+
+        `Hide venue content`
+            Hides the events time and location, if you just want to show the event content.
+        `Hide invite content`
+            Hides the invite content, if you just want to display the events venue details (time and date).
+
+- Newsletter
+    Display newsletter layout.
+
+        `Choose the amount of slots`
+            This option will allow you to pick how many article slots you want on the newsletter template.
+
+            Important! Refresh the page after applying the number of slots to see added blocks.
+
+- Other options
+    Display the staff spotlight
+        Show or hide the staff spotlight section.
+    Display the monthly links
+        Show or hide the monthly links section.
+        
+Editing the content
+-------------------
+Editing the content of the email is simple. Just point and click on the item you want to edit.
+
+Replacing  Images (Retina Images)
+~~~~~~~~~~~~~~~~~
+To make sure images are retina ready, you will need to disable Hubspots optimization.
```

Comment:
@KirMcC maybe explain in more detail that HubSpot optimizes images and that this can cause problems for retina support.